java/ql/src/Likely Bugs/Comparison/InconsistentCompareTo.qhelp

```diff
@@ -42,10 +42,9 @@ If you require a natural ordering that is inconsistent with <code>equals</code>,
   J. Bloch, <em>Effective Java (second edition)</em>, Item 12. Addison-Wesley, 2008.
 </li>
 <li>
-  Java API Documentation: 
-  <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Comparable.html#compareTo%28T%29">Comparable.compareTo</a>,
-  <a href="http://java.sun.com/javase/6/docs/api/java/lang/Comparable.html">Comparable</a>,
```

Comment:
Removed this link to `Comparable` because there is already a link to `Comparable#compareTo`.